src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,92 @@
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $white-color;
+
+  &__screen {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    width: 166px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 36px 0 4px;
+    text-align: left;
+  }
+
+  &__code {
+    font-size: 10px;
+    text-align: left;
+    color: $gray-color;
+    margin-top: 0;
+    margin-bottom: 16px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
```

Comment:
```suggestion
    margin-bottom: 16px;
    
    &-title {}
    &-price {}
```

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3530/files#r1306990381
didn't fix it
use parent selector